frontend/web/components/FeatureRow.js

```diff
@@ -310,14 +310,22 @@ class TheComponent extends Component {
                 <SegmentOverridesIcon
                   onClick={(e) => {
                     e.stopPropagation()
-                    this.editFeature(projectFlag, environmentFlags[id], 1)
+                    this.editFeature(
+                      projectFlag,
+                      environmentFlags[id],
+                      'segment-overrides',
```

Comment:
I thought that we had enums for these tab values now?

Reply:
We don't have enums for that one yet, I'll add some constants but probably would be more useful if that method was typed

Reply:
It should be good to go for now